.operations/writing-guidelines.polish.md

```diff
@@ -0,0 +1,31 @@
+# Nasz manifest pisania treści
+
+Jak zwiększamy komfort czytania i uczenia się dla naszych gości.
+
+## 1. Proste jest lepsze niż lepsze
```

Comment:
I think there should be simply: 'Proste jest lepsze'

Reply:
agree, fixed. done, thanks a lot!